### ISSUE TITLE
Problem with datetime using mixed charts (column and line)

### ISSUE THREAD

**Issue by andre-gasparin** [apexcharts/apexcharts.js#1798]
# Bug report

I am creating a mixed chart with line and columns using datatime on x-axis.

When the line has dots in dates and the column does not, the column does not skip the date that does not have.

if I change the column to row it will work perfectly again

## Explanation

![image](https://user-images.githubusercontent.com/55391441/88577998-3d0bc800-d01e-11ea-92de-3eb9f9ffdcd8.png)

Im using ApexCharts v3.19.3

**Comment by junedchhipa** [comment]
Can you create a [Codepen](https://codepen.io/apexcharts/pen/xYqyYm), please?

**Comment by andre-gasparin** [comment]
> Can you create a [Codepen](https://codepen.io/apexcharts/pen/xYqyYm), please?

sure, sorry!
https://codepen.io/andre-gasparin/pen/pogmJjx

u can use https://jsonformatter.curiousconcept.com/ in json, this json come from ajax.

**Comment by junedchhipa** [comment]
Thank you! Yeah, I can see the problem.
This will be addressed soon.

**Comment by Richcollu** [comment]
Hi @junedchhipa, 
has this problem been already solved?
Thank you!

**Comment by anderha** [comment]
**Fix: Adding the column type data before the line type data to the series array works for my usecase**

I had the same problem. @junedchhipa Seems to be some problem with the order of series elements. @andre-gasparin In your codepen try reversing the order of your array like this: `series: seriearray.reverse()` For me it produces the following graph, seems as expected, right?
![grafik](https://user-images.githubusercontent.com/36031262/101492483-33e41d80-3965-11eb-8ce6-c4b82a9ab952.png)

**Comment by junedchhipa** [comment]
Related to #2064 
This is fixed in this [commit](https://github.com/apexcharts/apexcharts.js/commit/9e99457aa3b5600800531b0c2e93be475aafa2df). A new version will be released soon with this fix.
